frontend/lib/ui/style/typography.css

```diff
@@ -65,6 +65,15 @@ p.md {
   line-height: 1.5rem;
 }
 
+div.md {
+  margin-bottom: 2rem;
+  width: 37.5rem;
+
+  &.ul-larger-padding ul {
+    padding-left: 1.75rem;
+  }
+}
+
```

Comment:
I feel that this is too specific to be in a generic (typography) css file. Maybe move this to a `ElectionReportPage.module.css`?

Reply:
Changed in 585ee39895bfe9b6b48e21cc4a27b3b86a81c199